ProcessLib/Process.h

```diff
@@ -184,6 +257,11 @@ class Process
 	std::unique_ptr<typename GlobalSetup::VectorType> _x;
 
 	AssemblerLib::SparsityPattern _sparsity_pattern;
+
+	std::vector<DirichletBc<GlobalIndexType>> _dirichlet_bcs;
+	std::vector<std::unique_ptr<NeumannBc<GlobalSetup>>> _neumann_bcs;
+
+	std::vector<ProcessVariable*> _process_variables;
```

Comment:
The vector of process variables is not owning. Therefore there is also no dtor here but in ProjectData. I'm adding a comment here.